src/integrationTest/java/uk/gov/hmcts/reform/userprofileapi/client/UserProfileRequestHandlerTest.java

```diff
@@ -40,7 +39,7 @@ public MvcResult sendPost(MockMvc mockMvc,
                 .headers(getMultipleAuthHeaders())
                 .content(jsonBody)
                 .contentType(APPLICATION_JSON))
-                .andExpect(status().is(expectedHttpStatus.value())).andReturn();
```

Comment:
any reason for removing this? Can you point the code where you are really asserting the http status value?

Reply:
As discussed yesterday, We should not validate status in sendPost method. Send post method will return response and we should add assertion after getting response from it

Reply:
thanks @shivajihmcts . That's why I asked the second question `Can you point the code where you are really asserting the http status value?` . Have you changed all the code which calling this method and added assertion?

Reply:
Also, can you please remove the variable `expectedHttpStatus` as it is no longer used

Reply:
@shivajihmcts - I think this conversation is still not resolved. I have above open questions

Reply:
@SujeethKumarAri That will be a big change better I will revert the removed status assertion and will create new ticket for it

Reply:
hi @shivajihmcts - I think that's what we decided yesterday to create a tech debt ticket a put it in the backlog as these are not as part of your current ticket :)